src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -163,6 +163,38 @@ object TezosTypes {
     )
   }
 
+  /**
+    * Representation of aggregation of votes (yay/nay/pass) for proposals
+    * in a given block, along with stake totals for the accounts that voted
+    * yay/nay/pass.
+    *
+    * @param timestamp     The timestamp when the calculation took place
+    * @param cycle         Cycle during which aggregation took place
+    * @param level         Block during which aggregation took place
+    * @param proposalHash  Hash of proposal voted on this block
+    * @param yayCount      Total count of yays for proposal
+    * @param nayCount      Total count of nays for proposal
+    * @param passCount     Total count of passes for proposal
+    * @param yayStake      Total stake of accounts who voted yay this block
+    * @param nayStake      Total stake of accounts who voted nay this block
+    * @param passStake     Total stake of accounts who voted pass this block
+    * @param totalStake    Total stake of accounts who voted
+    *
+    */
+  final case class VoteAggregates(
```

Comment:
I believe this file only contains types that reflect the types returned by the Tezos RPC. `VoteAggregates` is a 'synthetic' type so it should be placed in an appropriate file. As @ivanopagano pointed out, we need a case class instead of a type elsewhere where you have aggregated votes.

Reply:
Will move to a separate file.